packages/mui-material/src/Select/Select.test.js

```diff
@@ -1352,6 +1438,66 @@ describe('<Select />', () => {
         combinedStyle,
       );
     });
+
+    it('should be able to select the items on click', async () => {
+      const user = userEvent.setup();
+
+      render(
+        <Select defaultValue={[10]} multiple>
+          <MenuItem value={10}>Ten</MenuItem>
+          <MenuItem value={20}>Twenty</MenuItem>
+          <MenuItem value={30}>Thirty</MenuItem>
+        </Select>,
+      );
+
+      const trigger = screen.getByRole('combobox');
+
+      expect(trigger).to.have.text('Ten');
+
+      // open the menu
+      await user.click(trigger);
+
+      // const listbox = screen.queryByRole('listbox');
+      // expect(listbox).not.to.equal(null);
+
+      // const options = await screen.findAllByRole('option');
+      // Click second option
+      // await user.click(options[1]);
+
+      // expect(trigger).to.have.text('Ten, Twenty');
+
+      // Menu is still open in case of multiple
+      // expect(listbox).not.to.equal(null);
```

Comment:
@ZeeshanTamboli Is this used?

Reply:
Was testing `user-event` on CI. Updated now.